acme/challenge.go

```diff
@@ -101,6 +118,7 @@ type baseChallenge struct {
 	Validated time.Time `json:"validated"`
 	Created   time.Time `json:"created"`
 	Error     *AError   `json:"error"`
+	Retry     *Retry     `json:"retry"`
```

Comment:
I don't know if we need to return this. 8555 says to include retry info in the error field.